### ISSUE TITLE
security issues

### ISSUE THREAD

**Issue by dncVinny** [abrahampo1/ROAM#29]
any account you back up to the cloud is in his database as plain text 😑 😒 😐

**Comment by abrahampo1** [comment]
Nope, when you setup ROAM, it will ask you to create a "Crypto Password", so your accounts will be hashed and, if you want, The hash can be cloud stored.

All is protected with your crypto password.

**Comment by gnokemo** [comment]
> Nope, when you setup ROAM, it will ask you to create a "Crypto Password", so your accounts will be hashed and, if you want, The hash can be cloud stored.
> 
> All is protected with your crypto password.

nigger
